types/lesgo/exceptions/index.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./LesgoException";
```

Comment:
Since the original declaration was an `export default`, you need to explicitly export it just like in the source.

```ts
import LesgoException from './LesgoException';
export { LesgoException };
```

Reply:
Ahh right yeah makes total sense!